contracts/scripts/README.md

```diff
@@ -30,24 +22,14 @@ env CONTRACT_NAME=L1ETHGateway npx hardhat run --network $layer1 scripts/deploy_
 env CONTRACT_NAME=L1MessageQueue npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
 env CONTRACT_NAME=L2GasPriceOracle npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
 env CONTRACT_NAME=EnforcedTxGateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-
-# npx hardhat --network $layer1 run scripts/deploy_proxy_admin.ts
-# npx hardhat --network $layer1 run scripts/deploy_whitelist.ts
-# env CONTRACT_NAME=L1MessageQueue npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=L2GasPriceOracle npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# npx hardhat --network $layer1 run scripts/deploy_scroll_chain.ts
-# env CONTRACT_NAME=L1ScrollMessenger npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=L1GatewayRouter npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=L1StandardERC20Gateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=L1CustomERC20Gateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=L1ERC721Gateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=L1ERC1155Gateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=L1ETHGateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=EnforcedTxGateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
-# env CONTRACT_NAME=L2TxFeeVault npx hardhat run --network $layer1 scripts/deploy_predeploys.ts
-# env CONTRACT_NAME=L1WETHGateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
+#env CONTRACT_NAME=L1WETHGateway npx hardhat run --network $layer1 scripts/deploy_proxy_contract.ts
 
 # deploy contracts in layer 2, note: l2_messenger is predeployed
+
+env CONTRACT_NAME=L1GasPriceOracle npx hardhat run --network $layer2 scripts/deploy_predeploys.ts
```

Comment:
pre contract can't be deployed like this? Otherwise there will be problems with startup.